contributing/README.md

```diff
@@ -136,8 +136,8 @@ tox -e py36 -- verticapy/tests
 tox -e py37 -v -- verticapy/tests
 
 # Run an individual test on specified python versions.
-# e.g.: Run the test `test_vDF_append` under `test_vDF_combine_join_sort.py` on the python versions 3.7 and 3.8
-tox -e py37,py38 -- verticapy/tests/vDataFrame/test_vDF_combine_join_sort.py::TestvDFCombineJoinSort::test_vDF_append
+# e.g.: Run the test `test_append` under `test_join_union_sort.py` on the python versions 3.7 and 3.8
+tox -e py37,py38 -- verticapy/tests_new/core/vdataframe/test_join_union_sort.py::TestJoinUnionSort::test_append
```

Comment:
change python version pls. We support 39 or greater.

Reply:
I have updated python version to 39 or greater in README.me